airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -346,7 +342,11 @@ def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
 
     def parse_response(self, response: requests.Response, stream_slice: Mapping[str, Any] = None, **kwargs) -> Iterable[Mapping]:
         for record in response.json():  # GitHub puts records in an array.
-            yield self.transform(record=record, organization=stream_slice["organization"])
+            yield self.transform(record=record, stream_slice=stream_slice)
+
+    def transform(self, record: MutableMapping[str, Any], stream_slice: Mapping[str, Any]) -> MutableMapping[str, Any]:
```

Comment:
Same comment about the code re-usage for `Organisation` - dependent streams, we should reuse this code part.